TLSharp.Tests/TLSharpTests.cs

```diff
@@ -117,12 +123,18 @@ private void GatherTestConfiguration()
             {
                 user = await client.MakeAuthAsync(NumberToAuthenticate, hash, code);
             }
+            catch (CloudPasswordNeededException ex)
```

Comment:
I actually looking forward to find easier way to interact with user to get password but now it's best way